docker/dockerfile-build-ubuntu-rock

```diff
@@ -32,11 +32,10 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-ins
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
-# The version of MessagePack supplied in Ubuntu 16.04 is old and won't work,
-# so we'll manually download and install the DEB files
-RUN curl -O http://ftp.us.debian.org/debian/pool/main/m/msgpack-c/libmsgpack-dev_3.0.1-3_amd64.deb \
-         -O http://ftp.us.debian.org/debian/pool/main/m/msgpack-c/libmsgpackc2_3.0.1-3_amd64.deb
-RUN dpkg -i libmsgpack-dev_3.0.1-3_amd64.deb libmsgpackc2_3.0.1-3_amd64.deb
+# The version of MessagePack supplied in Ubuntu 16.04 is old and won't work
+# We need to add a repository that has a suitable version of MessagePack and download from there
+RUN echo "deb http://ftp.debian.org/debian stable main" >> /etc/apt/sources.list && \
```

Comment:
I apologize. What I suggested is worse than the curl/dkpg combination. If you add this to apt, there may be many other packages that are updated by mistake by the end user. Best to stick to the curl/dkpg.

I do not know the details of our support policy but in my opinion it is hard to call Ubuntu 16.04 a supported Tensile OS with this type of workaround.

Reply:
If this is only used for Docker, that shouldn't cause that big of an issue. I don't see why a user would update any packages on the container.

Reply:
That is correct. What are we going to suggest for users outside of docker?

Reply:
If they use rocBLAS, the installation script will automatically install the latest msgpack on their machines. Otherwise, I don't know how reasonable it will be to expect them to have msgpack installed.

Reply:
Since people often examine how we set things up in docker and mimic what we do there, the curl/dpkg combination is certainly easier for users to follow.  Updating /etc/apt/sources.list is a bit drastic IMO.  @saadrahim what's your take on this?

@wooky, if Ubuntu 16.04 users outside of docker do not have a recent enough libmsgpack-dev and libmsgpackc2 installed, what exactly will break?  Other than updating the Tensile wiki, what else can we do to help Ubuntu 16.04 users cope with this situation?